openmdao/utils/options_dictionary.py

```diff
@@ -44,6 +44,23 @@ def __init__(self, parent_name=None, read_only=False):
         self._parent_name = parent_name
         self._read_only = read_only
 
+    def __getstate__(self):
+        """
+        Return state as a dict.
+
+        Returns
+        -------
+        dict
+            State to get.
+        """
+        state = self.__dict__.copy()
+        user_options = state['_dict'].copy()
+        for i in user_options.copy():
```

Comment:
`i` by convention is used for integer looping variables.  I would use `name` or `key` here.